_layouts/candidate.html

```diff
@@ -34,11 +34,10 @@ <h2>{{ candidate.name }}</h2>
             <div class="candidate__expenditure-ceiling">
               This candidate has agreed to the voluntary spending limits.
               The maximum contribution this candidate can accept is $800.
-              <a target="_blank" href="https://www.oaklandca.gov/services/boards-and-commissions-index/ethics/election-campaign-rules-and-disclosure"
-                 alt="Voluntary spending limits - more information">
-                <img src="{{ "/assets/images/icon_more_info.png" | relative_url }}"
+              <span class="hover-info-container"><img src="{{ "/assets/images/icon_more_info.png" | relative_url }}"
                      alt="Voluntary spending limits - more information" />
-              </a>
+                     <span class="hover-info">For more detail on voluntary spending limits, see the <a href="{{ "/faq" | relative_url }}">FAQ article</a>.</span>
```

Comment:
Do you know where this is supposed to link to? Would you work with @sfdoran to figure that out?